### PR TITLE
Android fixes

### PR DESCRIPTION
**Pull request by larroy**

### CODE REVIEW

**Comment by larroy** [comment]
@KellenSunderland

**Comment by larroy** [comment]
Doxygen bug? https://github.com/doxygen/doxygen/commit/985faf287233badf65fa33d21bde17afa6970d60

**Comment by larroy** [comment]
@piiswrong please merge

File: mshadow/tensor.h [hunk 1]
```diff
@@ -1055,6 +1055,7 @@ inline void BatchGEMM(Tensor<Device, 3, DType> dst,
 #include "./tensor_gpu-inl.h"
 #include "./io.h"
 #include "./tensor_container.h"
+#include "./tensor_blob.h"
```
Comment (piiswrong) [inline mshadow/tensor.h]: Tblob has been moved to mxnet.
This tblob should be removed

**Comment by piiswrong** [comment]
please update

**Comment by larroy** [comment]
done, should be ready to merge. Removed tensor_blob.h

**Pull request merged**